packages/terra-button/src/_mixins.scss

```diff
@@ -8,7 +8,8 @@
   color: map-get($color-map, color);
   font-weight: map-get($color-map, font-weight);
 
-  &.is-focused {
+  &.is-focused,
+  &:focus {
```

Comment:
Solving the problem this way will force mouse selection to also get focus, which we don't want, so we'll need to try solving this another way.

Reply:
Fixed in 06a47fb35b2e4d69e5204e17572a246f8b1357a6